### DISCUSSION TITLE
Register a div tag in react hook form

### DISCUSSION THREAD

**Discussion by Joan-George** [react-hook-form/react-hook-form#9536] [Q&A]
Can I register HTML tags in the react hook form other than input fields?

If possible kindly provide some examples.

I'm trying to build a component that contains search and filters the result in the form from the group of div tags and the div tags should act as an input field.

**Comment by bluebill1049** [comment]
I think it's better to build a controlled component then register on the div, not sure it will work on top of my head.

**Comment by dmitrikar713** [comment]
Have you managed to register on div? Having the same problem rn...
